cilium/cmd/policy_trace.go

```diff
@@ -108,6 +124,33 @@ func parseAllowedSlice(slice []string) ([]string, error) {
 	return inLabels, nil
 }
 
+func parseL4RulesSlice(slice []string) ([]*models.Port, error) {
+	rules := []*models.Port{}
+	for _, v := range slice {
+		vSplit := strings.Split(v, "/")
+		if len(vSplit) != 2 {
+			return nil, fmt.Errorf("invalid format %q. Should be <port>/<protocol>", v)
```

Comment:
We should default to none as we do with policy. If not specified, both TCP and UDP will match.